SpatialGDK/Source/SpatialGDK/Private/SimulatedPlayers/SimulatedPlayer.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SimulatedPlayer.h"
+
+#include "Engine.h"
```

Comment:
See if you can get away with not including the whole `Engine.h`. `EngineMinimal.h` maybe.

Reply:
Nevermind, include is no longer necessary after refactor